pkg/skaffold/debug/transform.go

```diff
@@ -178,6 +178,11 @@ func transformManifest(obj runtime.Object, retrieveImageConfiguration configurat
 // transformPodSpec attempts to configure a podspec for debugging.
 // Returns true if changed, false otherwise.
 func transformPodSpec(metadata *metav1.ObjectMeta, podSpec *v1.PodSpec, retrieveImageConfiguration configurationRetriever) bool {
+	// skip annotated podspecs — allows users to customize their own image
```

Comment:
Shd we also add this to debug docs that users can use the "debug.cloud.google.com/config" to skip skaffold transforming pods?

Reply:
Good idea.

Reply:
I'll do this separately as there are a few other tweaks to the debug docs.